coreutils: Fix typo in flag

jonjohnsonjr opened the pull request:
This properly disables kill and uptime, which means we should no longer conflict with procps.

developer-guy commented:
relates to that: https://github.com/chainguard-images/images-private/actions/runs/6868160277/job/18678086296?pr=1463#step:15:81

developer-guy commented:
TLDR;

> installing apk packages: installing packages: installing procps: unable to install files for pkg procps: writing header for \"usr/bin/kill\": conflicting file for \"usr/bin/kill\" has no tar entry"

jonjohnsonjr committed:
coreutils: Fix typo in flag

This properly disables kill and uptime, which means we should no longer
conflict with procps.

Signed-off-by: Jon Johnson <jon.johnson@chainguard.dev>
